medusa/subtitle_providers/subtitulamos.py

```diff
@@ -0,0 +1,236 @@
+                        episode_url = self.server_url + 'episodes/{}'.format(episode_data['id'])
+                        logger.info('%s: Episode url found with year %s', self.__class__.__name__.upper(), episode_url)
+                        return episode_url
+            # attempt series without year
+            elif sanitize(series) in title:
+                for episode_data in result['episodes']:
+                    if season == episode_data['season'] and episode == episode_data['number']:
+                        episode_url = self.server_url + 'episodes/{}'.format(episode_data['id'])
+                        logger.info('%s: Episode url found without year %s', self.__class__.__name__.upper(), episode_url)
+                        return episode_url
+
+        return episode_url
+
+    def query(self, series, season, episode, year=None):
+        # get the episode url
+        episode_url = self._search_url_titles(series, season, episode, year)
+        if episode_url is None:
+            logger.warning('%s: No episode url found for %s, season %d, episode %d', self.__class__.__name__.upper(), series, season, episode)
+            return []
+
+        r = self.session.get(episode_url, headers={'Referer': self.server_url}, timeout=10)
+        r.raise_for_status()
+        soup = ParserBeautifulSoup(r.content, ['lxml', 'html.parser'])
+
+        # get episode title
+        logger.info('%s: Getting episode title', self.__class__.__name__.upper())
+
+        title_pattern = re.compile('{}x{:02d} - (.+)'.format(season, episode))
+        title = title_pattern.search(soup.select('.episode-name')[0].get_text().strip().lower()).group(1)
+
+        logger.info('%s: Episode title found: "%s"', self.__class__.__name__.upper(), title.upper())
+
+        subtitles = []
+        for sub in soup.find_all('div', attrs={'id': 'progress_buttons_row'}):
+            # read the language
+            language = Language.fromsubtitulamos(sub.find_previous('div', class_='subtitle_language').get_text().strip())
+            hearing_impaired = False
+
+            # modify spanish latino subtitle language to only spanish and set hearing_impaired = True
+            # because if exists spanish and spanish latino subtitle for the same episode, the score will be
+            # higher with spanish subtitle. Spanish subtitle takes priority.
+            if language == Language('spa', 'MX'):
+                language = Language('spa')
+                hearing_impaired = True
+
+            # read the release subtitle
+            release = sub.find_next('div', class_='version_name').get_text().strip()
+
+            # ignore incomplete subtitles
+            status = sub.find_next('div', class_='subtitle_buttons').contents[1]
+            if status.name != 'a':
+                logger.info('%s: Ignoring subtitle in [%s] not finished', self.__class__.__name__.upper(), language)
```

Comment:
It's not very clear why the status.name should be `a`. Please explain what is going on here and improve the logger message.